CHANGELOG.md

```diff
@@ -1,8 +1,61 @@
-## 0.25.0 — (unreleased)
+## 0.25.0 - January 17th, 2018
 
-No changes yet.
+### Consumer groups
 
+Consumer groups allow you to run groups of consumers together, rather than running them
+all at once in a single process. You define groups in your config file, and then specify
+a `--only-group` option when starting up Hutch with `hutch`.
 
+Contributed by Nickolai Smirnov.
+
+GitHub pull request: [#296](https://github.com/gocardless/hutch/pull/296)
+
+### Fix configuring Hutch with a URI
+
+When Hutch is configured to connect to RabbitMQ with a URI, we should respect the
+`amqps` specification, defaulting to the standard protocol ports when not specified.
+
+This means, for example, that `amqp://guest:guest@127.0.0.1/` connects to the server on
+port 5672 and does not use TLS, whereas `amqps://guest:guest@127.0.0.1/` connects to the
+server on port 5671 and uses TLS.
+
+This behaviour was introduced in [#159](https://github.com/gocardless/hutch/pull/159) but
+broken since then. This fixes it, and includes tests.
+
+Contributed by Michael Lennox.
```

Comment:
Hey there,

Could this be changed to Michael Canden-Lennox please?

Changed name last year but forgot to update it on github display name.

Thanks 😄